e2e/tests/block/k8s/create_block_test.go

```diff
@@ -0,0 +1,243 @@
+	require.Contains(s.T(), sout, "storageclass \"rook-block\" created", "Make sure storageclass is created")
+	require.Empty(s.T(), serr, "Make sure there are no errors")
+	require.Equal(s.T(), 0, status)
+
+	//make sure storageclass is created
+	present, err := s.kh.IsStorageClassPresent("rook-block")
+	require.Nil(s.T(), err)
+	require.True(s.T(), present, "Make sure storageclass is present")
+
+	//create pvc
+	sout1, serr1, status1 := s.pvcOperation("test-claim1", "create")
+	require.Contains(s.T(), sout1, "persistentvolumeclaim \"test-claim1\" created", "Make sure pvc is created")
+	require.Empty(s.T(), serr1, "Make sure there are no errors")
+	require.Equal(s.T(), 0, status1)
+
+	//check status of PVC
+	require.True(s.T(), s.isPVCBound("test-claim1"))
+
+	//check block image count
+	b, _ := s.bc.BlockList()
+	require.Equal(s.T(), s.init_blockCount+1, len(b), "Make sure new block image is created")
+	s.T().Log("Test Passed")
+
+}
+
+// Test case when persistentvolumeclaim is created for a valid storage class twice
+func (s *K8sBlockImageCreateSuite) TestCreateSamePVCTwice() {
+
+	s.T().Log("Test PVC(create block images) when storage class is not created")
+	s.T().Log("Test creating PVC(block images) when storage class is created")
+	//create pool and storageclass
+	sout, serr, status := s.storageClassOperation("test-pool-1", "create")
+	require.Contains(s.T(), sout, "pool \"test-pool-1\" created", "Make sure test pool is created")
+	require.Contains(s.T(), sout, "storageclass \"rook-block\" created", "Make sure storageclass is created")
+	require.Empty(s.T(), serr, "Make sure there are no errors")
+	require.Equal(s.T(), 0, status)
+
+	//make sure storageclass is created
+	present, err := s.kh.IsStorageClassPresent("rook-block")
+	require.Nil(s.T(), err)
+	require.True(s.T(), present, "Make sure storageclass is present")
+
+	//create pvc
+	sout1, serr1, status1 := s.pvcOperation("test-claim1", "create")
+	require.Contains(s.T(), sout1, "persistentvolumeclaim \"test-claim1\" created", "Make sure pvc is created")
+	require.Empty(s.T(), serr1, "Make sure there are no errors")
+	require.Equal(s.T(), 0, status1)
+
+	//check status of PVC
+	require.True(s.T(), s.isPVCBound("test-claim1"))
+
+	b1, _ := s.bc.BlockList()
+	require.Equal(s.T(), s.init_blockCount+1, len(b1), "Make sure new block image is created")
+
+	//Create same pvc again
+	sout2, serr2, status2 := s.pvcOperation("test-claim1", "create")
+	require.Empty(s.T(), sout2, "Make sure there are no output")
+	require.Contains(s.T(), serr2, "persistentvolumeclaims \"test-claim1\" already exists", "Check Error")
+	require.Equal(s.T(), 0, status2)
+
+	//check status of PVC
+	require.True(s.T(), s.isPVCBound("test-claim1"))
+
+	//check bock image count
+	b2, _ := s.bc.BlockList()
+	require.Equal(s.T(), len(b1), len(b2), "Make sure new block image is created")
+	s.T().Log("Test Passed")
+
+}
+
+func (s *K8sBlockImageCreateSuite) TearDownTest() {
+
+	s.pvcOperation("test-claim1", "delete")
```

Comment:
sure, I was going to use different names for each test, but ended up using same name to make clean up easier.